Restyle fix various small typos (#2508)
iterative/dvc.org: content/docs/start/data-and-model-versioning.md

# Get Started: Data Versioning

> How cool would it be to make Git handle arbitrary large files and directories

Reviewer: ```suggestion
How cool would it be to make Git handle arbitrarily large files and directories
```

> with the same performance that you get with small code files? Imagine doing a

Reviewer: ```suggestion
with the same performance it has with small code files? Imagine doing a
```
